packages/trader/src/AppV2/Utils/trade-params-utils.tsx

```diff
@@ -251,3 +253,43 @@ export const getOptionPerUnit = (unit: string, show_tick_from_5: boolean): { val
 
     return [[]];
 };
+
+export const getProposalRequestObject = ({
```

Comment:
is there a way to make this more generic? @ahmadtaimoor-deriv needs it in duration too.

it seems this will be more widely use due to the nature of our new application.

Reply:
Okay, will refactor